feature-crowdloan-api/src/main/java/jp/co/soramitsu/feature_crowdloan_api/data/repository/CrowdloanRepository.kt

```diff
@@ -35,6 +42,30 @@ class ParachainMetadata(
     val description: String,
     val rewardRate: BigDecimal?,
     val website: String,
-    val customFlow: String?,
-    val token: String
+    val token: String,
+    val flow: ParachainMetadataFlow?,
+) {
+    val isMoonbeam: Boolean
+        get() = name.toLowerCase(Locale.getDefault()) == "moonbeam"
+    val isAstar: Boolean
+        get() = name.toLowerCase(Locale.getDefault()) == "astar"
+    val isAcala: Boolean
+        get() = name.toLowerCase(Locale.getDefault()) == "acala"
+}
+
+class ParachainMetadataFlow(
+    val name: String?,
+    val data: ParachainMetadataFlowData?
 )
+
+class ParachainMetadataFlowData(
```

Comment:
I still do not see customisation for each flow, there definitely should not be one unified object for all cases